templates/cloudformation/aws_apollo_agent.yaml

```diff
@@ -128,57 +138,6 @@ Conditions:
     - !Ref 'CloudAccountId'
     - '590183797493'
 Resources:
-  Storage:
```

Comment:
instead of maintaining two definitions for this Storage resource, the `aws_data_store.yaml` stack can be deployed alongside the `aws_apollo_agent.yaml` stack